docker/dev.yml

```diff
@@ -0,0 +1,55 @@
+
+version: "3.7"
```

Comment:
i could see how running the prod docker-compose was probably causing some heartache, so i went ahead & made a dev one. this will be nice for folks moving forward, so now none of us are responsible for managing the DBs on new computers